manifmaker/client/helpers-events/task/TaskListComponent.js

```diff
@@ -5,31 +5,170 @@ class TaskListComponent extends BlazeComponent {
 
     events() {
         return [{
-            "change #task-list-team-selector": this.filterTeam
+            "keyup #search_task_name": this.filterName,
+            "click #advanced-search-button": this.switchAdvanced,
         }];
     }
 
+    /**
+     * Switch between hiding and showing the advanced search menu
+     * @param event
+     */
+    switchAdvanced(event){
+        if(this.isSearchAdvanced()) {
+            //TODO this is not the way to do it, it has to be done with Blaze and reactive var, I can show you
+            document.getElementById("advanced-search-button").innerHTML='More <i class="mdi mdi-chevron-down mdi-inline"></i>';
```

Comment:
this.isSearchAdvanced() return un boolean, tu peux donc utiliser cette function directement dans un bloc #if de blaze avec
```
{{#if isSearchAdvanced}}
```
Tu peux ainsi mettre ton template (ce que tu as dans innerHTML) directement dans les bloc if/else de Blaze. La function "switchAdvanced" se contente de modifier la reactive var taskListAdvancedSearch (ligne 24). 

Ainsi toute la logique métier est dans le component JS et tout le templating est dans le template du component. 

Dis moi si tu n'as tout compris.